secrets/amq-app-secret.json

```diff
@@ -3,7 +3,8 @@
     "apiVersion": "v1",
     "metadata": {
         "annotations": {
-            "description": "Examples that can be installed into your project to allow you to test the A-MQ template. You should replace the contents with data that is more appropriate for your deployment."
+            "description": "Examples that can be installed into your project to allow you to test the A-MQ template. You should replace the contents with data that is more appropriate for your deployment.",
+            "openshift.io/display-name": "Red Hat JBoss A-MQ (App Secret)"
```

Comment:
Secrets don't need this. Moreover, these secrets are only for convenience purposes. They aren't used by templates.